### PR TITLE
[DOC] Add hadoop and CDH version

### PR DESCRIPTION
**Pull request by qiyuangong**

### CODE REVIEW

File: docs/readthedocs/source/doc/UserGuide/hadoop.md [hunk 1]
```diff
@@ -38,8 +38,12 @@ You can run Analytics Zoo programs on standard Hadoop/YARN clusters without any
   export HADOOP_CONF_DIR=the directory of the hadoop and yarn configurations
   ```
 
+Hadoop version: Hadoop >= 2.7 or CDH 5.X, Hadoop 3.X or CHD 6.X are not recommended
+
 ### **1.1 Setup for CDH**
 
+CDH Version: CDH 5.X, CDH 6.X is not recommended
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/UserGuide/hadoop.md:45]: CDH Version: CDH 5.X; CDH 6.X is not supported

File: docs/readthedocs/source/doc/UserGuide/hadoop.md [hunk 2]
```diff
@@ -38,8 +38,12 @@ You can run Analytics Zoo programs on standard Hadoop/YARN clusters without any
   export HADOOP_CONF_DIR=the directory of the hadoop and yarn configurations
   ```
 
+Hadoop version: Hadoop >= 2.7 or CDH 5.X, Hadoop 3.X or CHD 6.X are not recommended
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/UserGuide/hadoop.md:41]: Move this to the beginning of the document.

File: docs/readthedocs/source/doc/UserGuide/hadoop.md (hunk 1 above)
Comment (qiyuangong) [reply docs/readthedocs/source/doc/UserGuide/hadoop.md]: Addressed

File: docs/readthedocs/source/doc/UserGuide/hadoop.md (hunk 2 above)
Comment (qiyuangong) [reply docs/readthedocs/source/doc/UserGuide/hadoop.md]: Move `Hadoop version: Hadoop >= 2.7 or CDH 5.X, Hadoop 3.X or CHD 6.X are not supported` to the beginning

**Pull request merged**